### PR TITLE
Cache fragment metadata for workspaces on the cloud

### PR DESCRIPTION
**Pull request by nalinigans**
Cache fragment metadata for workspaces on the cloud and add GitHub actions to test scripts in examples on azurite.

### CODE REVIEW

**Comment by nalinigans** [comment]
@mlathara, wondering if you re-review this? Thanks.

File: examples/genomicsdb_common.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        if all_intervals:
+            intervals.append(contig["name"])
+    return contigs_map, intervals
+
+
+interval_pattern = re.compile(r"(.*):(.*)-(.*)|(.*):(.*)|(.*)")
+
+
+# get tiledb offsets for interval
+def parse_interval(interval: str):
```
Comment (mlathara) [inline examples/genomicsdb_common.py:66]: I dunno if this is better or worse than regex -- but maybe we could consider using string `split` (plus check for existence of the delimiter). Might be simpler

**Review by mlathara — APPROVED** [review]
minor comment, feel free to merge without addressing it

File: examples/genomicsdb_common.py (hunk 1 above)
Comment (nalinigans) [reply examples/genomicsdb_common.py:66]: This is done just while parsing intervals for the cli. I don't expect much time spent on this function. Guess using split is simpler.

**Pull request merged**